app/views/events/_material_area.html.erb

```diff
@@ -27,7 +27,9 @@
       <% end %>
       </tbody>
     </table>
-
+  </div>
+  <div class="spacer-60"></div>
+  <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6 col-xs-offset-0 col-sm-offset-3 col-md-offset-3 col-lg-offset-3" id="event_material_area">
```

Comment:
Remove everything except for `col-xs-12 col-sm-6 col-xs-offset-0 col-sm-offset-3`